dbos/_dbos.py

```diff
@@ -324,6 +324,14 @@ def __init__(
         self.conductor_websocket: Optional[ConductorWebsocket] = None
         self._background_event_loop: BackgroundEventLoop = BackgroundEventLoop()
 
+        if (
+            "application_version" in config
+            and config["application_version"] is not None
```

Comment:
Which one have a higher priority on DBOS Cloud? DBOSConfig or env variables?

Reply:
Good call, updated it to not apply in the cloud